main/inc/lib/chat.lib.php

```diff
@@ -244,6 +244,26 @@ public function sanitize($text)
         $text = str_replace("\n", "<br>", $text);
         return $text;
     }
+    
+    public static function  set_disable_chat($status)
+    {
+            $_SESSION['disable_chat'] = $status;
+    }
+    
+    public function disable_chat()
+    {
+         if (isset($_SESSION['disable_chat'])) {
+             if (!empty($_SESSION['disable_chat'])){
```

Comment:
"!empty" already includes "isset", so you don't need to put both. Just if (!empty).
Space missing between )) and {